static/src/stylesheets/head.hosted-amp.scss

```diff
@@ -12,6 +12,8 @@
 @import 'amp/_type';
 @import 'amp/_fonts';
 @import 'amp/_helpers';
+@import 'amp/_content';
+@import 'amp/_social';
```

Comment:
makes sense that this is needed! Did you figure out why this wasn't failing locally?